definitions/private-law/json/AuthorisationCaseField/EditAndApprove/AuthorisationCaseField.json

```diff
@@ -677,5 +677,154 @@
           "CRUD": "R"
         }
       ]
-    }
+    },
+    {
+        "LiveFrom": "05/12/2023",
+        "CaseTypeID": "PRLAPPS",
+        "CaseFieldID": "checkTheOrderLabel",
+        "AccessControl": [
+          {
+            "UserRoles": [
+              "caseworker-privatelaw-courtadmin",
+              "caseworker-privatelaw-judge",
+              "caseworker-privatelaw-la",
+              "caseworker-privatelaw-systemupdate",
+              "caseworker-wa-task-configuration",
+              "citizen",
+              "caseworker-privatelaw-readonly"
+            ],
+            "CRUD": "R"
+          }
+      ]
+    },
+    {
+        "LiveFrom": "05/12/2023",
+        "CaseTypeID": "PRLAPPS",
+        "CaseFieldID": "openOrderAndReviewContentLabel",
+        "AccessControl": [
+          {
+            "UserRoles": [
+              "caseworker-privatelaw-courtadmin",
+              "caseworker-privatelaw-judge",
+              "caseworker-privatelaw-la",
+              "caseworker-privatelaw-systemupdate",
+              "caseworker-wa-task-configuration",
+              "citizen",
+              "caseworker-privatelaw-readonly"
+            ],
+            "CRUD": "R"
+          }
+      ]
+    },
+    {
+      "LiveFrom": "05/12/2023",
+      "CaseTypeID": "PRLAPPS",
+      "CaseFieldID": "whatToDoWithOrderSolicitor",
+      "AccessControl": [
+        {
+          "UserRoles": [
+            "caseworker-privatelaw-courtadmin",
+            "caseworker-privatelaw-systemupdate",
+            "caseworker-privatelaw-superuser",
+            "caseworker-privatelaw-la",
+            "caseworker-privatelaw-judge"
+          ],
+          "CRUD": "CRU"
+        },
+        {
+          "UserRoles": [
```

Comment:
Any reason for not using idam roles? if we are doing it here then it has to be consistent accross solicitor journey